client/src/lib/components/Icons/Bell/Bell.tsx

```diff
@@ -12,3 +15,28 @@ export const BellIcon: IconType = ({fill = COLORS.BLACK}) => (
     />
   </Icon>
 );
+
+type BellAnimatedProps = Omit<AnimatedIconProps, 'source'>;
+
+export const BellIconAnimated: React.FC<BellAnimatedProps> = forwardRef<
```

Comment:
Shouldn't the type be `AnimatedIconType` instead of this custom?

Reply:
Na, that does not type check